holoviews/element/path.py

```diff
@@ -193,22 +205,36 @@ class Ellipse(BaseShape):
 
     height = param.Number(default=1, doc="The height of the ellipse.")
 
-    aspect= param.Number(default=1.0, doc="The aspect ratio of the ellipse.")
+    width = param.Number(default=1, doc="The width of the ellipse.")
+
+    aspect= param.Number(default=1.0, doc="""
+       Final divisor only applied to the final height.""")
 
```

Comment:
Oh, I'm not arguing for the code to change, only for the docstring to reflect what the code does.